contracts/DEXes/SpiritJoint.sol

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "../Hedges/HedgilJoint.sol";
+
+interface ISpiritMasterchef is IMasterchef {
+    function pendingSpirit(uint256 _pid, address _user)
+        external
+        view
+        returns (uint256);
+}
+
+contract SpiritJoint is HedgilJoint {
```

Comment:
Should we move this to v2 too? Or later?

Reply:
done